dataland-frontend/src/components/forms/parts/elements/basic/MultiSelectFormElement.vue

```diff
@@ -72,6 +83,22 @@ export default defineComponent({
       type: String,
       default: "",
```

Comment:
I think this default value will be unnecessary/deleted once you use the `FormFIeldProps` from `FormFieldProps.ts`, because there it is already set to `short` as default.
Same goes for the `SingleSelectFormElement`.  I think there we also have an unnecessary default value.
 I hope this is won't break the look of those UI elements.

Reply:
This is not an issue anymore since `MultiSelectFormElement.vue` has no hardcoded props anymore.  They are all imported by `DropdownOptionFormFieldProps`.

But please in `MultiSelectFormField.vue` remove the hardocded `inner-class="short"` in the DOM for `<MultiSelectFormElement>`. This is unnecessary, because the setting `inner-class="short"` is already the default in the `DropdownOptionFormFieldProps`.

Reply:
done

Reply:
👍